backend/controllers/appointments/addCourtAppointment.js

```diff
@@ -8,23 +8,36 @@ const { serviceUnavailableMessage } = require('../../common-messages')
 
 const addCourtAppointmentsFactory = (elite2Api, logError) => {
   const getValidationMessages = fields => {
```

Comment:
I think this now duplicates what I did for the previous PR, the one with the isNumber function. It it worth reusing anything?